inc/python

```diff
@@ -52,7 +52,8 @@ function get_python_exec_prefix {
     if is_fedora || is_suse; then
         echo "/usr/bin"
     else
-        echo "/usr/local/bin"
+        #echo "/usr/local/bin"
+        echo "/usr/bin"
```

Comment:
I am really curious why this?  This value is used for install locations and pointing at the system install is dangerous, even for something as carefree as DevStack.

Reply:
haha, same feeling, but this is the reality.
There are also several other location issues like this, in neutron/glance configs.

Reply:
I am curious as to why... is /usr/local forbidden?  These are things that are _not_ managed by the OS packaging system, why must they mingle with those?  That is basic sysadmin 101 stuff...